LICENSE

```diff
@@ -0,0 +1,504 @@
+                  GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
hm, did you mean to add a license?

Reply:
you can obviously remove it. I tried to put a license in every project, but I do not remmber why I chose this one.

Reply:
so you want me to just make changes to the branch? Normally, this would bother the PR author. Unless you are sure you are done with the change.

Reply:
If possible, yes, I will then merge back my fork.
Yes, I am done.